src/displayapp/screens/Alarm.h

```diff
@@ -0,0 +1,53 @@
+/*  Copyright (C) 2021 mruss77, Florian
+
+    This file is part of InfiniTime.
+
+    InfiniTime is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    InfiniTime is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "Screen.h"
+#include "systemtask/SystemTask.h"
+#include "../LittleVgl.h"
+#include "components/alarm/AlarmController.h"
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Screens {
+      class Alarm : public Screen {
+      public:
+        Alarm(DisplayApp* app, Controllers::AlarmController& alarmController);
+        ~Alarm() override;
+        void SetAlerting();
+        void OnButtonEvent(lv_obj_t* obj, lv_event_t event);
+
+      private:
+        bool running;
+        uint8_t alarmHours = 0;
+        uint8_t alarmMinutes = 0;
```

Comment:
These get initialised to the AlarmController values in the constructor, so setting them to 0 here can be confusing.

Reply:
Fixed